tools/ratbagctl.in

```diff
@@ -719,7 +719,7 @@ parser_def = [
                     func: func_led_get,
                 },
                 {
-                    of_type: set,
+                    of_type: switch,
```

Comment:
any of the LED ones, I just could not make the color ones work, either it crashed on a missing `func` (hence the func: hunks) or it couldn't detect the `event4` node. I think it was `ratbagctl color set 12345 event4` or so